extensions/widgets/segmented/notes/20633.md

```diff
@@ -0,0 +1 @@
+#   [20633] Make corner radius of segmented widget a user settable property
```

Comment:
@jameshale I've got a feeling our release notes builder is going to freak out if there's more than one space between `#` and `[`. It's pretty pedantic ;-)